content/en/blog/2020/new-deployment-model/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: A New Istiod Deployment Model
+subtitle: Clear Separation between Mesh Operator and Mesh Users
+description: A new deployment model for Istio.
+publishdate: 2020-08-24
+attribution: "Lin Sun (IBM), Iris Ding (IBM)"
+keywords: [istiod,deployment model,central istiod,install,deploy,'1.7']
+---
+
+## Background
+
+From experience working with various service mesh users and vendors, we believe there are 3 key personas for a typical service mesh:
+
+* Mesh Operator, who manages the service mesh installation and upgrade.
+
+* Mesh Admin/Users:
+
+  1. Platform Owner, who owns the service mesh platform and defines the overall strategy and implementation for service owners to adopt service mesh.
+  1. Service Owner, who owns one or more services.
+
+It is common to have all these personas work on the same clusters without any clear separation.  For example, there are multiple ways to deploy Istio according to the [docs](/docs/setup/install/), which all start with mesh operator, platform owner and service owner sharing the single cluster first and then gradually expanding the mesh to multiple clusters or VMs.  None of these provided a clear separation between mesh operator and platform/service owner at the boundary of a cluster.  You may be thinking you could set up [Kubernetes RBAC rules](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) and namespaces to control which personas can do what within the cluster, however, sometimes you need stronger separation between the Istio control plane and the rest at the cluster level.  We're introducing a new deployment model for Istiod which enables mesh operators to install and manage the mesh control plane on dedicated clusters, separated from the data plane clusters.  This new deployment model could easily enable Istio vendors to run Istio control plane for service mesh users while users can focus on their workloads and Istio resources without worrying about installing or managing the Istio control plane. This model enables vendors to build managed Istio offerings that automatically upgrade Istio control plane for mesh users.
+
```

Comment:
I think the last sentence is the only important one. Who else, other than managed Istio providers would configure Istio this way. IMO, a simpler way to describe this whole thing is a way to run a `managed control plane` in a Kubernetes cluster (i.e., `control plane cluster : a Kubernetes cluster running a managed control plane`). Talking about a `Mesh Operator` persona managing the installation and upgrade of Istio is just using new words to say that the control plane is managed separately (a `managed control plane`). Why make it more complicated than that?

Reply:
Some feedback on naming from https://istio.slack.com/archives/C6KA8TTSS/p1598301755064400 on environments slack channel is that managed control plane often indicated run by cloud provider... and it is too narrow.  It doesn't have to be cloud provider, it can be a separate team focusing on operating istio control plane.

Reply:
I think you can greatly simplify this background:

Istio currently requires the control plane to run in one of the clusters in the mesh, leading to a lack of separation between the mesh operator and the mesh admin.

Also the final paragraph is not background, its an overview of what the blog is about.

Maybe just rename this section to overview and delete all but the last paragraph, and introduce the concepts of mesh operator and mesh admin in that paragraph.

We're introducing a new deployment model for Istio which enables a separation of the role of a mesh operator (manages the mesh control plane installation and operation) and mesh admin (manages the mesh configuration and membership). This new deployment model allows istiod to be used as an external control plane (link to glossary entry).

Reply:
Ok, I've removed most texts related to single cluster today and updated it as overview.

Reply:
Updated text looks good to me.

Reply:
I still get lost with "mesh operators" and "mesh admins"; my head blurs them into the same persona. Maybe refer to them without both starting with "mesh" ?

Reply:
If we remove mesh, say operators or admins, I think it is too wide and can cause confusions as well :-(